ruby-challenges.rb

```diff
@@ -19,12 +33,39 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def nums_arrays(array_of_nums)
+    array_of_nums.sum
+end
+p nums_arrays(nums_array1)
+p nums_arrays(nums_array2)
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
+class Bike
+    attr_accessor :model, :wheels, :current_speed
+    def initialize(model, wheels = 2, current_speed = 0)
+      @model = model
+      @wheels = wheels
+      @current_speed = current_speed
+    end
+    def get_info
+      "The #{model} bike has #{wheels} wheels and is going #{current_speed} mph."
+    end
+    def pedal_faster(increase_speed)
+        @current_speed = @current_speed + increase_speed
+        p @current_speed
+    end
+    def break(decrease_speed)
+        @current_speed = @current_speed - decrease_speed
+        p [@current_speed, 0].max
+    end
+end
+  
+my_bike = Bike.new('Trek')
+p my_bike.get_info
```

Comment:
It is best practice to put the `p` on the method call rather than the inner logic.